tests/test_loan_transitions.py

```diff
@@ -54,6 +55,40 @@ def test_loan_request(db, params):
     assert loan['state'] == 'PENDING'
 
 
+def test_loan_extend(loan_created, db, params,
```

Comment:
I think we are testing more than aspects in this test, could be split in more.

Reply:
I think it is better to have one test for the extension even if we test two aspects of it. Otherwise we have to reinit the loan a second time to test.

Reply:
it would be really helpful to put some comments in order to follow the scenario that is tested. Just for the future us to be able to understand really quick the test.

Reply:
@zzacharo Yes this will be very helpful and I think this will be added with the task **Write docstrings and documentation #30** https://github.com/inveniosoftware/invenio-circulation/issues/30

Reply:
The  two cases should be tested `from_end_date == True` and `from_end_date == False`. It can be in two different test functions.